docs/source/tutorial/components/detail_ui_setup_panel.mdx

```diff
@@ -60,11 +60,6 @@ And update it to match the following:
 
 ![constant 0, priority 1000, multiplier 1:1](../images/aspect_ratio_multiplier.png)
 
-<<<<<<< HEAD
->**Note**: You may still see some sizing errors at this point, but those will go away in the next step.
```

Comment:
We should keep this note, otherwise people may be like "WTF is going on here" when it has a million IB warnings.

Reply:
ok cool this was me messing up conflict resolution